app.go

```diff
@@ -2,12 +2,12 @@ package main
 
 import (
 	"fmt"
+	"github.com/aquasecurity/bench-common/util"
```

Comment:
Please use [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports) so you get the conventional import ordering